src/DDTrace/Tracer.php

```diff
@@ -63,6 +64,11 @@
      */
     private $scopeManager;
 
+    /**
+     * @var ScopeInterface
+     */
+    private $rootSpan;
```

Comment:
Being it a scope, shouldn't it be named `$rootScope`, and so the getter name also should change.
What  do you think?  :)